stories/atoms/Menu.stories.tsx

```diff
@@ -0,0 +1,153 @@
+            <DropdownMenu.Portal>
+                <DropdownMenu.Content>
+                    <MultiSelectMenuItem
+                        onCheckedChange={() => {
+                            handleOnChange('all');
+                        }}
+                        key="all"
+                        isSelected={isAllSelected()}
+                        variant="select-all"
+                    >
+                        Select all
+                    </MultiSelectMenuItem>
+                    {generalValues.map((value) => (
+                        <MultiSelectMenuItem
+                            key={value}
+                            isSelected={selectedValues.includes(value)}
+                            onCheckedChange={() => {
+                                handleOnChange(value);
+                            }}
+                        >
+                            {value}
+                        </MultiSelectMenuItem>
+                    ))}
+                    <MultiSelectMenuItem
+                        key="group1"
+                        isSelected={isGroup1Selected()}
+                        variant="group-title"
+                        onCheckedChange={() => {
+                            handleOnChange('group1');
+                        }}
+                    >
+                        Options
+                    </MultiSelectMenuItem>
+                    {group1.map((value) => (
+                        <MultiSelectMenuItem
+                            key={value}
+                            isSelected={selectedValues.includes(value)}
+                            onCheckedChange={() => {
+                                handleOnChange(value);
+                            }}
+                        >
+                            {value}
+                        </MultiSelectMenuItem>
+                    ))}
+                    <DropdownMenu.Sub>
+                        <DropdownMenu.SubTrigger>Items →</DropdownMenu.SubTrigger>
+                        <DropdownMenu.Portal>
+                            <DropdownMenu.SubContent>
+                                {group2.map((value) => (
+                                    <MultiSelectMenuItem
+                                        key={value}
+                                        isSelected={selectedValues.includes(value)}
+                                        onCheckedChange={() => {
+                                            handleOnChange(value);
+                                        }}
+                                    >
+                                        {value}
+                                    </MultiSelectMenuItem>
+                                ))}
+                            </DropdownMenu.SubContent>
+                        </DropdownMenu.Portal>
+                    </DropdownMenu.Sub>
+                </DropdownMenu.Content>
+            </DropdownMenu.Portal>
+        </DropdownMenu.Root>
```

Comment:
We also can wrap items in another component the selection/navigations still works. Super!